release-calendar/test/test-data.ts

```diff
@@ -32,13 +32,12 @@ export default async function addTestData(
 ): Promise<void> {
   await Promise.all(
     [
-      new Release('1234567890123'), // lts
-      new Release('2234567890123'), // TODO(estherkim): promote to stable
-      new Release('3234567890123'), // TODO(estherkim): promote to percent_*
-      new Release('4234567890123'), // TODO(estherkim): promote to opt_in_*
-      new Release('5234567890123'), // nightly
-      new Release('6234567890123'),
-    ].map(async (release) => {
+      new Release('1234567890123'), // nightly
+      new Release('2234567890123'), // lts
+      new Release('3234567890123'), // perc-beta and perc-experimental
+      new Release('4234567890123'), // stable
+      new Release('5234567890123'), // opt-in-beta and opt-in-experimental
+    ].map(async release => {
```

Comment:
Keep the same order as the comments had originally, because `1...` is older than `2...` and reflects our release process.

Reply:
Originally releases were being saved to the repository and then called back using .getReleases. getReleases does not preserve the order that releases were added. In order to avoid having to resort and to not call releases one at a time with .getRelease(name), I pushed the creations and promotions to the repository all at the same time.